src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.navigation {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+    align-items: end;
```

Comment:
The value 'end' is not a valid value for 'align-items'. It should be 'flex-end' to align the items at the end of the container.